podpac/core/data/interpolate.py

```diff
@@ -1212,16 +1219,45 @@ def interpolate(self, source_coordinates, source_data, eval_coordinates, output_
         # for debugging purposes, save the last defined interpolator queue
         self._last_interpolator_queue = interpolator_queue
 
+        
```

Comment:
The changes in the following two blocks enable incremental interpolation from multiple interpolators in the queue. The tricky bit is making intermediate datastructures and coordinates to store the intermediate results.